packages/components/src/Toggle/Toggle.component.js

```diff
@@ -34,7 +34,7 @@ function Toggle({ id, label, className, intermediate, ...props }) {
 
 	return (
 		<div
-			className={classNames('checkbox tc-toggle switch checkbox', theme['tc-toggle'], className, {
+			className={classNames('checkbox tc-toggle', theme['tc-toggle'], className || 'switch', {
```

Comment:
In the case of `Checkbox` component, this will result in `checkbox tc-toggle checkbox` classnames. Which is weird. iof "checkbox" is the default, shouldn't this component be `Checkbox` and `Switch` call this component with a `switch` classname?

Reply:
<img src="https://memegenerator.net/img/instances/48004726.jpg" alt="Too damn right">